backend/src/main/java/com/woowacourse/momo/group/domain/group/Group.java

```diff
@@ -42,13 +43,17 @@ public class Group {
     @Column(nullable = false)
     private String name;
 
-    @Column(nullable = false)
-    private Long hostId;
+    @ManyToOne
+    @JoinColumn(name = "host_id")
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn
```
@JoinColumn 을 사용한 엔티티에서 가지는 컬럼이라면 name 은 생략해도 됩니다 😊
그리고 @ManyToOne 의 기본 fetch 전략은 EAGER 이므로 LAZY 로 바꾸어주면 좋겠습니다~!ㅎㅎ

Reply:
`@JoinColumn`의 name내용은 제거하였습니다!! 
다만 host 연관관계의 fetch전략의 경우 참여하기를 제외한 Group관련 모든 기능들이 Host의 정보를 필요로 하기에 Lazy보다 Eager로 설정하는 것이 성능상에서 더 이점이 있지 않을까?하는 생각이 들어 fetch전략의 변경은 보류했습니다!
내일 한번 이야기해봐요😁